templates/firefox-ios.xliff

```diff
@@ -318,7 +318,7 @@
         <source>$(MOZ_PRODUCT_NAME)</source>
       </trans-unit>
       <trans-unit id="CFBundleShortVersionString">
-        <source>23.1</source>
+        <source>0.0.1</source>
```

Comment:
This doesn't look right?

Reply:
As we don't update version number on master branch we decided to make it constant 0.0.1 for master.

Hence why we have this change.